tests/e2e_tests/fluxcal_e2e.py

```diff
@@ -0,0 +1,74 @@
+# Flux calibration E2E Test Code
+
+import argparse
+import os, shutil
+import glob
+import pytest
+import numpy as np
+
+import corgidrp
+import corgidrp.data as data
+import corgidrp.mocks as mocks
+import corgidrp.walker as walker
+import corgidrp.detector as detector
+import corgidrp.fluxcal as fluxcal
+
+@pytest.mark.e2e
+def test_expected_results_e2e(tvacdata_path, e2eoutput_path):
+    #mock a point source image
+    fwhm = 3
+    star_flux = 1.5e-09 #erg/(s*cm^2*AA)
+    cal_factor = star_flux/200
+    flux_image = mocks.create_flux_image(star_flux, fwhm, cal_factor)
+    flux_dataset = data.Dataset([flux_image])
+    output_dir = os.path.join(e2eoutput_path, 'flux_sim_test_data')
+
+    if os.path.exists(output_dir):
+        shutil.rmtree(output_dir)
+    os.mkdir(output_dir)
+    
+ 
+    flux_dataset.save(output_dir, ['flux_e2e_{0}.fits'.format(i) for i in range(len(flux_dataset))])
+    flux_data_filelist = []
+    for f in os.listdir(output_dir):
+        flux_data_filelist.append(os.path.join(output_dir, f))
+    print(flux_data_filelist)
+
+    # make DRP output directory if needed
+    fluxcal_outputdir = os.path.join(e2eoutput_path, "l2b_to_fluxcal_factor_output")
+    if os.path.exists(fluxcal_outputdir):
+        shutil.rmtree(fluxcal_outputdir)
+    os.mkdir(fluxcal_outputdir)
+
+    ####### Run the DRP walker
+    print('Running walker')
+    walker.walk_corgidrp(flux_data_filelist, '', fluxcal_outputdir, template="l2b_to_fluxcal_factor.json")
+    
+        ####### Load in the output data. It should be the latest kgain file produced.
+    fluxcal_file = glob.glob(os.path.join(fluxcal_outputdir, '*FluxcalFactor*.fits'))[0]
+    flux_fac = data.FluxcalFactor(fluxcal_file)
+    print("used color filter", flux_fac.filter)
+    print("used ND filter", flux_fac.nd_filter)
+    print("fluxcal factor", flux_fac.fluxcal_fac)
+    print("fluxcal factor error", flux_fac.fluxcal_err)
+    assert flux_fac.fluxcal_fac == pytest.approx(cal_factor, abs = 1.5 * flux_fac.fluxcal_err)
+    
```

Comment:
can you update the `labels` dictionary in `caldb.py` to include the FluxcalFactor class so that this file is automatically added to the calibration DB?

As a consequence of doing this, you'll also need to do `caldb.remove_entry(flux_fac)` to clean-up the e2e test.